cylc/uiserver/resolvers.py

```diff
@@ -397,10 +472,31 @@ async def service(
                 executor=self.executor
             )
 
-        else:
+        elif command == 'play':
             return await Services.play(
                 workflows,
                 kwargs,
                 self.workflows_mgr,
                 log=self.log
             )
+
+        raise Exception()
+
+    async def subscription_service(
+        self,
+        info: 'ResolveInfo',
+        _command: str,
+        workflows: List[Tokens],
+        task=None,
+        file=None
+    ):
+        if file:
+            file = LOG_FILE_MAPPINGS.get(file, file)
+        async for ret in Services.cat_log(
+            workflows[0],
+            self.log,
+            info,
+            task,
+            file
+        ):
```

Comment:
Note: For 8.2.0 it might be worth adapting the interface of `cat-log` to allow it to accept full file names.

Reply:
Turns out it already does, sorted!